torchvision/prototype/transforms/_deprecated.py

```diff
@@ -28,6 +28,9 @@ def __init__(self) -> None:
         super().__init__()
 
     def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
+        # datumbox: why check here again for types? Can't we just define
+        # `_transformed_types = (PIL.Image.Image, np.ndarray)`
+        # and then eliminate this if/else?
```

Comment:
cc @vfdev-5 / @pmeier

Reply:
Probably legacy. @vfdev-5 Are you taking this or should I?

Reply:
@pmeier yes go ahead. I think it should safe to remove if/else here

Reply:
As discussed offline, I'll pick this up for all `_transformed_types`.

Reply:
#6494

Reply:
Done in #6494